examples/models/models_loading_gltf.c

```diff
@@ -43,11 +47,24 @@ int main(void)
     SetTargetFPS(60);                   // Set our game to run at 60 frames-per-second
     //--------------------------------------------------------------------------------------
 
+    unsigned int currentFrame = 0;
+
     // Main game loop
     while (!WindowShouldClose())        // Detect window close button or ESC key
     {
         // Update
         //----------------------------------------------------------------------------------
+        ModelAnimation anim = modelAnimations[animIndex];
+        if (IsKeyPressed(KEY_UP)) {
+            animIndex = (animIndex + 1) % animsCount;
+        }
+
+        if (IsKeyPressed(KEY_DOWN)) {
```

Comment:
Please, could you review the parentheses to be aligned, following raylib coding conventions?